packages/typescript/test/devmode_override/tsconfig-override.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        // Explicitly set types settings so typescript doesn't auto-discover types.
+        // If all types are discovered then all types need to be included as deps
+        // or typescript may error out with TS2688: Cannot find type definition file for 'foo'.
+        "types": []
+    },
+    "bazelOptions": {
+        "devmodeTargetOverride": "es5",
+        "devmodeModuleOverride": "amd"
```

Comment:
since amd is currently the default could we have another one to test that we can output say target=es5 module=esnext or module=commonjs

Reply:
umd is the default :)

Reply:
oh my mistake